src/smpdflib/core.py

```diff
@@ -719,6 +729,19 @@ def make_pdf_results(pdf, Q, flavors=None, xgrid=None):
     return results
 
 
+def make_pdfcorr_results(pdf, Q, flavors=None, xgrid=None):
```

Comment:
Do we need this function, or could we do something like make_pdf_results(...).correlations()?
Maybe this is better because make_pdf_results gives you one result per flavour...